docs/user-guide/filtering-layers.md

```diff
@@ -182,6 +182,10 @@ Once this filter is set, it is always possible to edit the coordinates and the d
 
 <img src="../img/filtering-layers/edit_geom.jpg" class="ms-docimage" style="max-width:500px;"/>
 
+!!! note
+     For [Dashboard](exploring-dashboards.md) [widgets](widgets.md) (charts, table and counter) the **Region of Interest** option is called **Area of interest**. With this type of filter the user can define a spatial filter without connecting the widget to the map widget. An example can be the following:
```

Comment:
```suggestion
     Also for [Dashboard](exploring-dashboards.md) [widgets](widgets.md) (charts, table and counter) it is possible to define a spatial filter without necessarily connect the widget to the map widget by using the usual **Area of interest** filtering section. The example below sows how:
```

Reply:
@ElenaGallo where **Region of Interest** appear in the UI? I see "Area of interest" everywhere. It seems the documentation need to be fixed.